tests/protractor/page-objects/forms/delivery-report.po.js

```diff
@@ -1218,21 +1188,27 @@ module.exports = {
 
   //note to CHW
   getNoteToCHW: () => {
-    return element(by.name('/delivery/group_note/g_chw_sms')).getText();
+    return element(by.css('textarea')).getAttribute('value');
   },
 
   //summary page
   getOutcomeText: () => {
     return element(by.css('[data-value=" /delivery/group_delivery_summary/display_delivery_outcome "]'))
-      .getText();
+      .getInnerHtml();
   },
 
   getDeliveryLocationSummaryText: () => {
     return element(by.css('[data-value=" /delivery/group_summary/r_delivery_location "]'))
-      .getText();
+      .getInnerHtml();
   },
 
   getFollowUpMessage: () => {
-    return element(by.css('[data-value=" /delivery/group_note/g_chw_sms "]')).getText();
+    return element(by.css('[data-value=" /delivery/group_note/g_chw_sms "]')).getInnerHtml();
+  },
+
+  deleteReport: (done) => {
+    element(by.css('li.read.selected')).getAttribute('data-record-id').then(function(docId) {
```

Comment:
Please be consistent with inline function declarations and use es6 notation, i.e.
```
docId => …
```
in place of
```
function(docId) {
```